packages/components/src/DateTimePickers/InputDateTimePicker/InputDateTimePicker.component.js

```diff
@@ -187,6 +199,52 @@ class InputDateTimePicker extends React.Component {
 		this.updateDateTime(date, time, fullString, errMsg);
 	}
 
+	onFocusInput() {
+		this.switchDropdownVisibility(true);
+	}
+
+	setContainerRef(ref) {
+		this.containerRef = ref;
+	}
+
+	setDropdownWrapperRef(ref) {
+		this.dropdownWrapperRef = ref;
+	}
+
+	mountDocumentHandler() {
+		document.addEventListener('click', this.documentHandler);
+		document.addEventListener('focus', this.documentHandler);
```

Comment:
I think the focus event on document is useless

Reply:
If we change focus with keyboard by navigating outside.
Or if we use a special keyboard combination for an other part of the page and change focus programmatically.

But whatever the use case, just by semantic : if the focus is done outside, it means we are no more focused on the picker :p

Reply:
Normally, all DOM events should go down in capture phase then up in bubble phase.
document --> target --> document
_If there propagation has not been stopped by a ninja._

I'm checking why the focus is not handled in document.